src/js/view/controls/controls.js

```diff
@@ -317,7 +317,7 @@ export default class Controls extends Events {
                     break;
                 case 13: // enter
                 case 32: // space
-                    if (document.activeElement.className === 'jw-switch' && evt.keyCode === 32) {
+                    if (document.activeElement.classList.contains('jw-switch') && evt.keyCode === 32) {
```

Comment:
evt.keycode is always 32 here, we can remove the extra check. Looks good after that!